tests/js/search/testDomain.js

```diff
@@ -118,6 +119,20 @@ function create_mock_shard_with_link_table (link_table_hash) {
     return mock_shard_file;
 }
 
+describe('Domain', function () {
+    describe('get_ekn_version', function () {
+        it('should throw an exception when datadir can\'t be found', function (done) {
+            let engine = new Engine.Engine();
```

Comment:
Sorry I didn't notice this the first time; `engine` isn't used, so can be removed.